packages/paste-design-tokens/tokens/global/sizing.yml

```diff
@@ -161,3 +161,35 @@ props:
   size-topbar:
     value: "{!size-topbar}"
     comment: "Sizing token for the minimum topbar height."
+
+  # base
+  size-base-0:
+    value: "{!size-base-0}"
+    comment: Generic sizing token scale for UI components
```

Comment:
suggestion: can we remove the word "scale" from these descriptions? I know you just copied it over from the existing ones but I think it makes more sense without it...